docs/examples/userguide/language_basics/casting_python.py

```diff
@@ -8,7 +8,7 @@ def main():
     # as the correct pointer type that is assigned to them.
     # They do not need to be typed explicitly.
 
-    ptr = cython.cast(cython.p_void, python_string)
+    ptr = cython.cast(cython.pointer(void), python_string)
     adress_in_c = cython.cast(Py_intptr_t, ptr)
     address_from_void = adress_in_c        # address_from_void is a python int
```

Comment:
```suggestion
    address_in_c = cython.cast(Py_intptr_t, ptr)
    address_from_void = address_in_c        # address_from_void is a python int
```

May as well fix some spelling here too

Reply:
thanks!